README.md

```diff
@@ -1,45 +1,122 @@
 
-### Installation
+## Contribution guidelines
+
+Contributions from the community are highly encouraged.
+
+To contribute please follow these steps.
+
+### Step 1: Fork and clone
+
+To ensure that each change is properly peer reviewed, we request that you follow standard open source contribution processes.
+
+This means that you should start by forking the project into your own GitHub account.
+
+Then, clone your own fork of the repo onto your local machine.
 
 ```
-$ yarn
+$ git clone git@github.com:<your-account>/documentation-legacy.git
 ```
 
-### Local Development
+### Step 2: Add upstream remote
+
+Then add the original repository as a new remote to your local clone.
 
 ```
-$ yarn start
+$ cd documentation-legacy
+$ git remote add upstream git@github.com/temporal/documentation-legacy.git
 ```
 
-This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
+### Step 3: Create a local branch
 
-### Build
+When you are ready to start making changes, sync your local clone with the upstream project and your forked GitHub project.
 
 ```
-$ yarn build
+$ git checkout master
+$ git pull upstream master && git push origin master
+```
+
+Then, create a branch to work on.
+
+```
+$ git checkout -b your-branch
+```
+
+Now you can make your changes. See the next section about building and viewing the site locally for those steps.
+
+### Step 4: Submit a pull request
+
+Once you are ready to submit your changes, create a pull request.
+
+```
+git push -u origin your-branch
+```
+
+This will create the branch on your GitHub project fork.
+
+**Note** that the -u flag only needs to be used the first time you create a pull request for this branch.
+
+Next, visit the project fork in your GitHub account and locate the new branch, and click "Compare and pull request".
+
+Ensure the "base fork" points to the correct repository and branch and provide a clear title and explanation of the changes with links to any issues that are being fixed by the change.
+
+Then click "Create pull request".
+
+## Building and viewing the site locally
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/).
+
+Check out their [installation requirements](https://v2.docusaurus.io/docs/installation#requirements) for tooling prerequisites.
+
+### Step 1: Initialize yarn
+
+The first step is to initialize the use of Yarn to build the site locally.
+
+From your terminal, change directory to the project root and run:
+
+```
+$ yarn
 ```
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
+### Step 2: Build the website
 
-### Versioning
+The following command generates static content into the `build` directory which can be served using any static contents hosting service.
 
 ```
-$ yarn run docusaurus docs:version 1.1.0
+$ yarn build
```

Comment:
You should never need to use this right?

Reply:
removing yarn build